generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -937,7 +937,7 @@ message GetNIRFSASessionRequest {
 
 message GetNIRFSASessionResponse {
   int32 status = 1;
-  nidevice_grpc.Session ni_rfsa_session = 2;
+  nidevice_grpc.Session nirfsa_session = 2;
```

Comment:
Was this the only thing wrong with spelling? or will there be more to the task>?

Reply:
Good question. Everything else was tracked in separate tasks, especially #465, which David has been working on (I think he's got all of the ones listed in the issue done; but keeps finding more stuff and it's been combined with work for different personalities).

I already did the manual proto review as AB#1781514, which added some entries to bug 465. Though I'll probably have to do at least one more pass and maybe I'll find something else.